test/e2e/container_probe.go

```diff
@@ -62,9 +62,10 @@ var _ = Describe("Probing container", func() {
 
 		isReady, err := podRunningReady(p)
 		expectNoError(err)
-		Expect(isReady).To(BeTrue())
+		Expect(isReady).To(BeTrue(), "pod should be ready")
```

Comment:
suggest some logging around the api.IsPodReady check, since that's where this usually times out and it often isn't clear what state the pod is in (the ready check is based off status.Conditions which is a list). You could then include the state here.